browser(webkit): use webkit generate-bundle tool to generate the minibrowser bundles

clopez opened the pull request:
This changes the `archive.sh `script to use the [webkit generate-bundle tool](https://bugs.webkit.org/show_bug.cgi?id=215266) to generate the MiniBrowser bundles instead of using our custom logic.

`pw_run.sh` is simplified because the new bundles include a wrapper script that sets the needed `LD_LIBRARY_PATH` environment variables.` pw_run.sh` now calls this wrapper script (for GTK or WPE) instead of calling the `MiniBrowser` binary directly when running from a bundle.

To test this you need to first update webkit to the last checkout:
1. Update playwright master to de5ecc028f40857c52a2edfc90fa37b80456a867 or above
2. Update the webkit checkout and rebuild webkit
```
browser_patches/prepare_checkout.sh webkit
browser_patches/webkit/build.sh
```
3. Generate the bundle with something like
```
browser_patches/webkit/archive.sh /tmp/test-new-bundle.zip
```
... and test it

//CC @aslushnikov

aslushnikov reviewed (approved):
Awesome!

Can we also bump the webkit build number to trigger build with the new build system? (simply advancing the `//browser_patches/webkit/BUILD_NUMBER` and optionally updating the signature)

The automated way to bump build would be to re-export webkit patch:

```sh
$ ./browser_patches/prepare_checkout.sh wk
$ ./browser_patches/export.sh wk
```

clopez committed:
browser(webkit): use webkit generate-bundle tool to generate the bundles

 * This changes the archive.sh script to use the webkit generate-bundle
 tool to generate the MiniBrowser bundles instead of using our custom
 logic.

 * pw_run.sh is simplified because the new bundles include a wrapper
 script that sets the needed LD_LIBRARY_PATH environment variables.
 pw_run.sh now calls this wrapper script (for GTK or WPE) instead of
 calling the MiniBrowser binary directly when running from a bundle.

clopez commented:
I just updated the `BUILD_NUMBER` file

aslushnikov committed:
Merge branch 'master' into webkit-use-generate-bundle
